packages/@aws-cdk/aws-servicecatalogappregistry/test/integ.application.ts

```diff
@@ -4,9 +4,26 @@ import * as appreg from '../lib';
 const app = new cdk.App();
 const stack = new cdk.Stack(app, 'integ-servicecatalogappregistry-application');
 
-new appreg.Application(stack, 'TestApplication', {
+const application = new appreg.Application(stack, 'TestApplication', {
   applicationName: 'myApplicationtest',
   description: 'my application description',
 });
 
+const attributeGroup = new appreg.AttributeGroup(stack, 'TestAttributeGroup', {
+  attributeGroupName: 'myAttributeGroupTest',
+  description: 'my attribute group description',
+  attributes: {
+    stage: 'alpha',
+    teamMembers: [
```

Comment:
How about adding some `boolean`, `numbers` and Map attributes here? To demonstrate that all of the various types work.

Reply:
good call, added here and also in `README`